web/src/package.json

```
@@ -130,6 +130,8 @@
     "@types/react-redux": "7.1.7",
     "@types/react-router": "5.1.4",
     "@types/react-router-dom": "5.1.3",
+    "@typescript-eslint/eslint-plugin": "^2.24.0",
+    "@typescript-eslint/parser": "^2.24.0",
```

```suggestion
    "@typescript-eslint/eslint-plugin": "2.24.0",
    "@typescript-eslint/parser": "2.24.0",
```

plus a rerun of `yarn install` after to update `yarn.lock`